DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,178 +1,176 @@
 {
-  "object": {
```

Comment:
Just a warning since I'd expect only BSK to change here.  Looks odd that so much is changing.

Reply:
This appears to be a side effect of the SwiftLint plugin, the tools version has changed so Xcode wants to use the new Package.resolved format. I thought this change was being made in the SwiftLint PR, but apparently not.

I'll look at fixing it here.